### DISCUSSION TITLE
Query Log Filter Grouping

### DISCUSSION THREAD

**Discussion by sunindersingh** [AdguardTeam/AdGuardHome#5372] [Q&A]
Hi,

Is it possible to group items in query log?

For example : There is 100 Block request for translate.googleapis.com and 17 Block request for apis.google.com. And in Query Log it shows total 117 Lines, but if there is someway to filter the log lines so it only show Group "Additional column" unique request logs. Like :-

Group Time Request Response Client
Count [100]  19:23:31 translate.googleapis.com Blocked x.x.x.x
Count [17]  19:23:31 apis.google.com Blocked x.x.x.x

So once we click on Group Column Count it expand. 

Ty.

**Comment by ammnt** [comment]
@sunindersingh, unfortunately no. But this is really good feature request. I like it👍🏼Better make a feature request for this.

**Reply by sunindersingh** [reply]
Ok, how to add feature request??
